src/Chapter2.hs

```diff
@@ -620,7 +636,10 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven [] = []
+takeEven (first' : second' : tail') = first' : takeEven tail'
+takeEven (first' : second') = first' : []
```

Comment:
You can slightly rearrange cases for clarity. The following has exactly the same behaviour as yours, but for readability it usually easier to see smaller and special cases first. Well, at least for me :slightly_smiling_face: 

```suggestion
takeEven [] = []
takeEven [first'] = [first']
takeEven (first' : second' : tail') = first' : takeEven tail'
```